packages/governance/src/committee.js

```diff
@@ -92,7 +93,12 @@ const start = (zcf, privateArgs) => {
                 return E(voteCap).submitVote(voterHandle, positions, 1n);
               };
 
-              return zcf.makeInvitation(continuingVoteHandler, 'vote');
+              return zcf.makeInvitation(
+                continuingVoteHandler,
```

Comment:
`continuingVoteHandler` is exercised by the Wallet. If it's ever not an `EmptyProposal`, I think that's under the user's control, but I have no idea how.